tephra-core/src/main/java/co/cask/tephra/TxConstants.java

```diff
@@ -310,4 +310,16 @@
       { DefaultSnapshotCodec.class, SnapshotCodecV2.class, SnapshotCodecV3.class, SnapshotCodecV4.class };
   }
 
+  /**
+   * Configuration for transaction log edit entries
+   */
+  public static final class TransactionLog {
+    /**
+     * Key used to denote the number of entries appended.
+     */
+    public static final String NUM_ENTRIES_APPENDED = "entries.append.count";
```

Comment:
Since we'll write this string for every sync, let's try to reduce its size. Maybe just call it `count`? Would it be confusing?